build/release-images.sh

```diff
@@ -30,6 +30,9 @@ if [[ "${KUBE_BUILD_CONFORMANCE}" =~ [yY] ]]; then
     CMD_TARGETS="${CMD_TARGETS} ${KUBE_CONFORMANCE_IMAGE_TARGETS[*]}"
 fi
 
+# TODO(dims): Remove this when we get rid of hyperkube image
+CMD_TARGETS="${CMD_TARGETS} cmd/kubelet cmd/kubectl"
```

Comment:
how did the 1.17 release build without this? cmd/kubelet isn't in KUBE_CONFORMANCE_IMAGE_TARGETS

Reply:
we use `make release` a slightly different code path: https://cs.k8s.io/?q=make.*release&i=nope&files=&repos=kubernetes/release

Reply:
is it a problem if cmd/kubectl is included in the list twice? it is also in KUBE_CONFORMANCE_IMAGE_TARGETS

Reply:
@liggitt tried both `KUBE_BUILD_CONFORMANCE=y` and `KUBE_BUILD_CONFORMANCE=n` the double invocation is harmless, i wanted to contain the change into a single line that we can remove later. can split it up if you wish. this target is developer convenience only.